topics/variant-analysis/tutorials/trio-analysis/tutorial.md

```diff
@@ -0,0 +1,441 @@
+>    - *"Match type"*: `case sensitive`
+>
+{: .hands_on}
+
+## Download files
+After the filtering you should have a tabular file with 3 lines each containing the ID of a VCF file from case 5, as shown below.
+```
+EGAF00005573839	1	33859685	53277d993a780e0c295079bf1346ee9f	Case5_IC.17.g.vcf.gz
+EGAF00005573866	1	33350163	7e612852ee0824be458fbf9aeecaa61a	Case5_M.17.g.vcf.gz
+EGAF00005573882	1	42856357	14b53924d1492e28ad6078ceb8cfdbc7	Case5_F.17.g.vcf.gz
+```
+
+> ### {% icon hands_on %} Hands-on: Download listed VCFs
+>
+> 1. {% tool [EGA Download Client](toolshed.g2.bx.psu.edu/repos/iuc/ega_download_client/pyega3/4.0.0+galaxy0) %} with the following parameters:
+>    - *"What would you like to do?"*: `Download multiple files (based on a file with IDs)`
+>        - {% icon param-file %} *"Table with IDs to download"*: `Filtered list of files` (output of **Search in textfiles** {% icon tool %})
+>        - *"Column containing the file IDs"*: `Column: 1`
+>
+>
+> After the download you should have a collection with one file for each family member, i.e. mother (M), father (F), and case (IC). Make sure the files are recognized as the **vcf_bgzip** format.
+>
+> {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %} 
+>
+{: .hands_on}
+
+# Pre-Processing
+Before starting the analysis, the VCF files have to be pre-processed in order to meet input requirements of the tools which we will use for the downstream analysis. 
+
+## Decompress VCFs
+First, we need to decompress our bgzipped VCFs, since we will use a text manipulation tool as a next step to process the VCFs. To decompress the vcf we will use a built-in tool from galaxy, which can be accessed by manipulating the file itself in a similair fashion as changing its detected type.
+
+> ### {% icon hands_on %} Hands-on: Convert compressed vcf to uncompressed.
+>
+> Open the collection of VCFs and execute the following steps for each VCF.
+>
+> 1. Click on the **pencil** {% icon galaxy-pencil %} of the vcf you want to convert.
+>
+> 2. Click on the **convert tab** {% icon galaxy-gear %}.
+>
+> 3. Under **Target datatype** select `vcf (using 'Convert compressed file to uncompressed.')`
+>
+> 4. Click {% icon exchange %} **Create Dataset**.
+>
+> After transforming all the VCFs you need to combine the converted VCFs into a colllection again.
+>
+> > ### {% icon tip %} Tip: Adding files to collection
+> >
+> > 1. Click on the dataset **selector icon** {% icon param-check %} in your history.
+> > 2. Select the 3 vcf files.
+> > 3. Click on the button **For all selected...** and select **Build Dataset List**
+> > 4. Give the new collection a name and click on **Create collection**.
+> >
+> {: .tip}
+>
+{: .hands_on}
+
+## Add chromosome prefix to vcf

```

Comment:
It's more for the bcftools norm step since it uses a reference genome with those prefixes. A way around it could be to use the original genome used to call the variants instead of the local one on the EU Galaxy. Would that be better?

Reply:
I see. Then I don't know a way around decompressing and adding the prefix either.
However I less complicated way is to specify the datatype as `vcf` in the *Upload data* dialog. This will prompt Galaxy to decompress the file as part of the upload process and store it in the history *only* in this form, which means one step less and less quota consumption.

Reply:
Oh, I didn't know that! I've added it in the tutorial but I still have to keep the decompress step when downloading the files from the EGA archive. 4cb441ff692dad48e6e2b2e1ea5bd6883e905c69